components/NavigationMenu/index.jsx

```diff
@@ -0,0 +1,50 @@
+const NavigationMenu = () => (
```

Comment:
As this component is only used in the Header, it could be located in a folder called SubComponents, within the Header folder, what do you think? This suggested structure is the one used in Quantum